examples/splunk-hec-metrics-with-telemetrygen/splunk.yml

```diff
@@ -0,0 +1,11 @@
+splunk:
+    conf:
+      indexes:
+        directory: /opt/splunk/etc/apps/search/local
+        content:
+          metrics:
+            coldPath: $SPLUNK_DB/metrics/colddb
```

Comment:
Where is this `$SPLUNK_DB` variable defined?

Reply:
https://splunk.github.io/docker-splunk/ADVANCED.html#create-custom-configs

Reply:
Note how this links to https://splunk.github.io/splunk-ansible/ADVANCED.html#inventory-script which unfortunately doesn't define $SPLUNK_DB, though it's used in the example given in the first link.

Reply:
Ah, from some research, that env var is set by Splunk itself and is probably set in the docker image.